<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Deity Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="followerTab">
       <attribute name="title">
        <string>Followers</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableWidget" name="followerTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Basic Info</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerName"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Class:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="followerClass">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Level:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerLevel"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>Exp:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerExperience"/>
               </item>
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>/</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerRequiredExperience">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Gold:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerGold"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Stats</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Str</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerStrength"/>
               </item>
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>End</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerEndurance"/>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Int</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerIntelligence"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Agl</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerAgility"/>
               </item>
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Wil</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerWillpower"/>
               </item>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Lck</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerLuck"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Pts</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerPoints"/>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Rep</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerReputation"/>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Dvt</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="followerDevotion"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="followerEquipment">
              <property name="text">
               <string>Equipment</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="followerInventory">
              <property name="text">
               <string>Inventory</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="followerQuests">
              <property name="text">
               <string>Quests</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="followerSave">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="locationTab">
       <attribute name="title">
        <string>Locations</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QTableWidget" name="locationTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Basic Info</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="locationName"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Type:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="locationType"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>X:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="locationX"/>
               </item>
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>Y:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="locationY">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <item>
                <widget class="QLabel" name="label_21">
                 <property name="text">
                  <string>Floors:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="locationFloors"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QPushButton" name="locationShop">
              <property name="text">
               <string>Shop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="locationDrops">
              <property name="text">
               <string>Drops</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="locationMonsters">
              <property name="text">
               <string>Monsters</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="locationSave">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="materialTab">
       <attribute name="title">
        <string>Materials</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="itemTab">
       <attribute name="title">
        <string>Items</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="craftingTab">
       <attribute name="title">
        <string>Crafting</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="monsterTab">
       <attribute name="title">
        <string>Monsters</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="lexiconTab">
       <attribute name="title">
        <string>Lexicon</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="questTab">
       <attribute name="title">
        <string>Quests</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
